src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -418,7 +422,7 @@ export const Participants = () => {
             sx={{ width: '32rem' }}
             data-testid={`${dataTestid}-search`}
           />
-          {appletId && (
+          {canAddParticipant && (
```

Comment:
Remember to add this to the empty state add participant button below as well

<img src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/1e209800-b7a0-43c5-87b5-8bfe00e2fb37" width="480">